vagrant_variables.yml.openstack

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
can the content of integrated into the default `vagrant_variables.yml.sample` instead?